packages/babel-traverse/test/scope.js

```diff
@@ -303,6 +303,15 @@ describe("scope", () => {
 
       expect(path.scope.bindings.a).toBe(path.get("body[0]").scope.bindings.a);
     });
+
+    it("references after re-crawling", function() {
+      const path = getPath("function Foo() { var jsx; }");
+
+      path.scope.crawl();
+      path.scope.crawl();
+
+      expect(path.scope.references.jsx).toBeTruthy();
```

Comment:
Note: I have no idea of what `references` on the `scope` means since it's clearly not related to `binding.referencePaths`. However, this PR aligns the post-crawl state to the pre-crawl one.

Reply:
I'm not sure if this is the full story but those references seem to be a simple `Record<name, true>` of everything that could clash with generated identifiers (or other nodes), since it's not possible to just check some single scope to be sure if some name can be considered "sage", but it rather requires full knowledge about the module/script